### DISCUSSION TITLE
Why not separate repos, are projects related by code?

### DISCUSSION THREAD

**Discussion by romanr** [nagadomi/nunif#204] [Q&A]
Is there a reason for having three projects as one repository? It makes more difficult to understand since root level readme describes all.  
and what does name "iw3" stand for? :)

**Comment by nagadomi** [comment]
Because common modules are used in all projects and I am developing them at the same time.
With separated repository I have to commit 4 times for one change. Doing that every day is very distressing.
So the reason for not separating the repositories is for my own advantage, not for the advantage of the users or other delopers. I am basically developing for myself.

iw3 stands for "I want to see 2D images as 3D images on a VR device" or a sentence with a similar meaning. The first version was iww3, but I changed it because it reminded people of war.
